crates/bevy_ecs/src/event.rs

```diff
@@ -63,6 +63,10 @@ enum State {
 /// Each event can be consumed by multiple systems, in parallel,
 /// with consumption tracked by the [`EventReader`] on a per-system basis.
 ///
+/// If no [ordering](https://github.com/bevyengine/bevy/blob/main/examples/ecs/ecs_guide.rs)
```

Comment:
There's a danger with linking to a file on main, but I don't have a good suggestion.

Reply:
```suggestion
/// If no [ordering](https://github.com/bevyengine/bevy/blob/latest/examples/ecs/ecs_guide.rs)
```

This will make sure we link to the latest release, rather than the examples on `main`.